syntax = "proto3";

package authPackage;

service AuthService {
    rpc validateCredentials(login_credentials) returns (AuthResponse);
}

message login_credentials {
    string username = 1;
    string password = 2;
}

message AuthResponse {
    int32 status = 1;
}